marimbabot_ur5_moveit_config/launch/gazebo.launch

```diff
@@ -25,8 +25,8 @@
   <!-- Load joint controller parameters for Gazebo -->
   <rosparam file="$(find marimbabot_ur5_moveit_config)/config/gazebo_controllers.yaml" />
   <!-- Spawn Gazebo ROS controllers -->
-  <node name="gazebo_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="joint_state_controller" />
+  <node name="gazebo_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="joint_state_controller gripper_controller arm_controller marimba_bars_controller" />
   <!-- Load ROS controllers -->
-  <include file="$(dirname)/ros_controllers.launch"/>
+  <!-- <include file="$(dirname)/ros_controllers.launch"/> -->
```

Comment:
Remove this code if it is not needed. It's saved in git.

Reply:
This whole file is not needed 😅 Will remove this now to avoid confusion (gazebo in filename), the rest of the extra files will be cleaned up in another PR, see: #105